components/form/KeyValue.vue

```diff
@@ -2,49 +2,41 @@
 import debounce from 'lodash/debounce';
 import { typeOf } from '@/utils/sort';
 import { removeAt, removeObject } from '@/utils/array';
-import { asciiLike } from '@/utils/string';
-import { base64Encode, base64Decode } from '@/utils/crypto';
+import { base64Encode, base64Decode, binarySize } from '@/utils/crypto';
 import { downloadFile } from '@/utils/download';
 import TextAreaAutoGrow from '@/components/form/TextAreaAutoGrow';
 import { get } from '@/utils/object';
 import Select from '@/components/form/Select';
 import FileSelector from '@/components/form/FileSelector';
 import { _EDIT, _VIEW } from '@/config/query-params';
-
```

Comment:
I think you might have an odd linting rule that puts these in. Generally I think we have lines where they have been removed in the file